demo_exam/Task01.cpp

```diff
@@ -0,0 +1,84 @@
+#include <iostream>

```

Comment:
At the actual exam you will have to write all your function in one .h file. You can test them in an additional .cpp file (but you won't have to upload it in moodle most likely).